protecode/scanning_util.py

```diff
@@ -157,6 +158,9 @@ def _image_ref_metadata(self, container_image, omit_version):
         if not omit_version:
             metadata_dict['IMAGE_REFERENCE'] = container_image.image_reference()
             metadata_dict['IMAGE_VERSION'] = container_image.version()
+            metadata_dict['IMAGE_DIGEST'] = container.registry.to_hash_reference(
```

Comment:
could you just store the digest (including the algorithm, i.d. everything behind the color (`:`))?
the full image reference (including the digest) could - as discussed yesterday - also be stored (better name required^^)
maybe DIGEST_IMAGE_REFERENCE?

also: how about adding a method to `ContainerImage`, that exposes the hashy-reference-value? this method could/should also apply caching
additional value-add: we could/should check whether the image-reference already contains a digest-tag, in which case no lookup would be required.